doc/reference/reference_lua/net_box.rst

```diff
@@ -602,7 +610,31 @@ Below is a list of all ``net.box`` functions.
             the result of a sync request, it is structured differently: as a
             table, instead of as the unpacked values.
 
-.. _net_box-triggers:
+    ..  _net_box-return_raw:
+
+    ..  method:: request(... {return_raw=...})
+
+        ``{return_raw=true}`` is ignored for
+        *   methods that return nil:
+            ``begin``, ``commit``, ``rollback``, ``upsert``, ``prepare``;
```

Comment:
```suggestion
        ``{return_raw=true}`` is ignored for:
        
        *   Methods that return ``nil``: ``begin``, ``commit``, ``rollback``, ``upsert``, ``prepare``
```

The asterisk now looks as, well, an asterisk.
Please refer to our [list punctuation style guide](https://www.tarantool.io/en/doc/latest/contributing/docs/style/#ending-punctuation-in-lists-and-tables). The rules in English and Russian are different.